packages/estatico-boilerplate/package.json

```diff
@@ -43,14 +43,17 @@
   },
   "dependencies": {
     "bows": "^1.7.0",
+    "custom-event-polyfill": "^1.0.6",
     "dom-delegate": "^2.2.0",
     "highlight.js": "^9.12.0",
     "jquery": "^3.3.1",
     "lodash": "^4.17.10",
     "merge-stream": "^1.0.1",
+    "nodelist-foreach-polyfill": "^1.2.0",
```

Comment:
You can think about using https://github.com/WebReflection/dom4.  It includes these polyfills and some more. We normally load this in a separate entry file either in a sync way using `document.write` or in [a ordered way async](https://github.com/boffinHouse/rawblock-kik/blob/gh-pages/source/assets/js/_inlinehead-behavior.js#L25-L27).